deployment.go

```diff
@@ -109,6 +110,43 @@ var deploymentStatusCmd = &cobra.Command{
 	Requires the Inertia daemon to be active on your remote - do this by 
 	running 'inertia [REMOTE] up'`,
 	Run: func(cmd *cobra.Command, args []string) {
+		config, err := client.GetProjectConfigFromDisk()
+		if err != nil {
+			log.WithError(err)
+		}
+		remotename := strings.Split(cmd.Parent().Use, " ")[0]
+		remote, found := config.GetRemote(remotename)
+		if !found {
+			println("No such remote " + remotename)
+			println("Inertia currently supports one remote per repository")
+			println("Run `inertia remote -v' to see what remote is available")
+			os.Exit(1)
+		}
+
+		host := "http://" + remote.GetIPAndPort()
+		remoteResp, err := http.Get(host)
```

Comment:
We don't need this `http.Get` either. Have a look at the code for [`deployment.Status()` - notice it makes an HTTP request to the daemon already](https://github.com/ubclaunchpad/inertia/blob/master/client/deployment.go#L85). The following code is around [line 155](https://github.com/ubclaunchpad/inertia/pull/135/files#diff-4b475da4d1c32296f086ca8abdb29c83R155) of this revision:

```go
# makes an HTTP request to daemon
resp, err := deployment.Status()
if err != nil {
    # oh no! request failed
    log.WithError(err)
}
```

If this operation fails, we know the daemon is not online. If it succeeds, we know the daemon is online, and we can report that the daemon is listening at x address 🎉 

Speaking of which, there is a bug somewhere here that would be awesome if you could dig it up - the `status` command currently breaks with a null pointer error if the daemon is offline (#70) 😢 if you can find it you can close two tickets at once 🏋️

Reply:
If you do find that second bug that @bobheadxi mentioned, perhaps consider applying the fix through a separate commit to make reviewing easier?

Reply:
Heads up that I actually (partially) fixed this in the branch I am working on (`rob/auth` or something) 😅 

Sorry but that's another massive, over-scoped PR coming 😞

Reply:
The problem was that `log.WithError()` doesn't stop execution, so the program ends up using things like `resp` from the above example, which of course is `nil` because an `err` was returned.